### ISSUE TITLE
Cannot see comments on files of a pull request 

### ISSUE THREAD

**Issue by liach** [jonan/ForkHub#133]
If someone comments on the diff of the pr, you can see it on dashboard but cannot navigate into it. You cannot also see the comments when you visit the pull request directly.

**Comment by alexfornuto** [comment]
I would love to see this issue resolved.

**Comment by liach** [comment]
Is this not available due to the limit of Github api?

**Comment by jonan** [comment]
Not really, it's just the lack of time to implement it... :disappointed:
